.dockerignore

```
@@ -0,0 +1,7 @@
+__pycache__
+.github
+.pytest_cache
```

Annoyingly, despite the obvious similarity with the `.gitignore` file, `.dockerignore` doesn't work _quite_ the same. In particular, while `.gitignore` assumes you want to ignore the named files/folders whenever they appear at any level, `.dockerignore` only ignores from the root folder by default. I actually got caught out here by your tests picking up the previous `__pycache__` files after I tweaked something and rebuilt because of this

To fix that we can explicitly say that we want to ignore these at any level with `**/<name>`:
```suggestion
**/__pycache__
.github
**/.pytest_cache
```

---

Ah weird, thanks!